### ISSUE TITLE
Is this library out of date?

### ISSUE THREAD

**Issue by SeqCrafter** [picklelo/reflex-chat#3]
When using in my project, it raised an error:

```
File "/Users/panxiaoguang/Desktop/SequenceTool/SequenceTool/.venv/lib/python3.12/site-packages/reflex_chat/components.py", line 46, in action_bar
    rx.input.root(
    ^^^^^^^^^^^^^
AttributeError: 'TextField' object has no attribute 'root'
```

**Comment by psecoros** [comment]
> When using in my project, it raised an error:
> 
> ```
> File "/Users/panxiaoguang/Desktop/SequenceTool/SequenceTool/.venv/lib/python3.12/site-packages/reflex_chat/components.py", line 46, in action_bar
>     rx.input.root(
>     ^^^^^^^^^^^^^
> AttributeError: 'TextField' object has no attribute 'root'
> ```

I am recieving the same error, did you find a solution ? i think that it is becasue the reflex version and reflex-chat version are incompatible as long as the reflex-chot do not update to the 0.8 reflex version

**Comment by SeqCrafter** [comment]
> > When using in my project, it raised an error:
> > ```
> > File "/Users/panxiaoguang/Desktop/SequenceTool/SequenceTool/.venv/lib/python3.12/site-packages/reflex_chat/components.py", line 46, in action_bar
> >     rx.input.root(
> >     ^^^^^^^^^^^^^
> > AttributeError: 'TextField' object has no attribute 'root'
> > ```
> 
> I am recieving the same error, did you find a solution ? i think that it is becasue the reflex version and reflex-chat version are incompatible as long as the reflex-chot do not update to the 0.8 reflex version

You are true, the best choice is to wrap yourself Chat library using this repo as the reference.
